components/core/Routes/User.py

```diff
@@ -171,10 +99,7 @@ def update_user_request(username):
         try:
             newUser = User(data['user_name'], data['password'], int(data['auth']), data['email'])
             status = update_user(newUser, username)
-            if status == "success":
-                resp = Response(response='{"status": "'+ status + '"}', status=200)
-            else:
-                resp = Response(response='{"error":"' + status + '"}', status=400)
+            resp = Response(response='{"status":"'+ status + '"}', status= (200 if status == "success" else 400))
```

Comment:
This changes the response structure, 
Previously they were
```
{ "status" : 200 }
```

```
{ "error" : 400 }
```

But your code changes it to,
```
{ "status" : 200 }
```
and
```
{ "status" : 400 }
```

However, I like it that way. But I guess that would break the frontend.

We can make this streamlined in later tasks.